app/models/event.rb

```diff
@@ -3,6 +3,8 @@ class Event < ApplicationRecord
   belongs_to :user
   belongs_to :currency
 
+  has_one :pool
```

Comment:
`dependent: :destroy`?

Reply:
Hmm, I’m not sure. When an event is destroyed we want more things to happen than just destroying the associated pool.